src/lib/__tests__/ReleaseHelper.ts

```diff
@@ -0,0 +1,153 @@
+import { ReleaseHelper } from 'lib/ReleaseHelper';
+
+describe('ReleaseHelper', () => {
```

Comment:
For validating if the release logic is correct, please look through these tests.